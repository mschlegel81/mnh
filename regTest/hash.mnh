hex(i)->format('%16x',i).replace(' ','0');

fileHash(filename:string)->fileExists(filename) ? fileContents(filename).hash.hex : '--file does not exist--';
main->print(fileHash(myPath));
main(...)->printf("%X%\t%X%",$params,each(p,$params,fileHash(p)));
