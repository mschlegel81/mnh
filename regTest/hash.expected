D54ED7F309F5DD78
